### PR TITLE
Add support for testing async QuickInfo providers

### PR DESCRIPTION
**Pull request by mhutch**

### CODE REVIEW

**Comment by mhutch** [comment]
@garuma would appreciate if you could release a new nupkg with this :)

File: Microsoft.VisualStudio.MiniEditor/Microsoft.VisualStudio.MiniEditor.csproj [hunk 1]
```diff
@@ -6,7 +6,6 @@
     <DefaultItemExcludes>$(DefaultItemExcludes);vs-editor-api\**\*</DefaultItemExcludes>
     <BaseEditorPath>vs-editor-api\src\Editor\</BaseEditorPath>
     <SignAssembly>true</SignAssembly>
-    <DelaySign>false</DelaySign>
```
Comment (garuma) [inline Microsoft.VisualStudio.MiniEditor/Microsoft.VisualStudio.MiniEditor.csproj]: Was that needed?

File: Microsoft.VisualStudio.MiniEditor/Microsoft.VisualStudio.MiniEditor.csproj [hunk 2]
```diff
@@ -220,48 +232,48 @@
     <EmbeddedResource Include="$(BaseEditorPath)Core\Impl\ContentType\Strings.resx">
       <Generator>ResXFileCodeGenerator</Generator>
       <ManifestResourceName>Microsoft.VisualStudio.CoreUtilityImplementation.ContentType.Strings</ManifestResourceName>
-      <LastGenOutput>$(BaseEditorPath)Core\Impl\ContentType\Strings.Designer.cs</LastGenOutput>
+      <LastGenOutput>vs-editor-api\src\Editor\Core\Impl\ContentType\Strings.Designer.cs</LastGenOutput>
```
Comment (garuma) [inline Microsoft.VisualStudio.MiniEditor/Microsoft.VisualStudio.MiniEditor.csproj]: Same for those, is it to make VSMac happy?

**Review by garuma — APPROVED** [review]
Thanks for the contribution! Just two group of csproj changes I had questions on but otherwise seems good to me

File: Microsoft.VisualStudio.MiniEditor/Microsoft.VisualStudio.MiniEditor.csproj (hunk 1 above)
Comment (mhutch) [reply Microsoft.VisualStudio.MiniEditor/Microsoft.VisualStudio.MiniEditor.csproj]: Nope, VSMac must have done that 😞

File: Microsoft.VisualStudio.MiniEditor/Microsoft.VisualStudio.MiniEditor.csproj (hunk 2 above)
Comment (mhutch) [reply Microsoft.VisualStudio.MiniEditor/Microsoft.VisualStudio.MiniEditor.csproj]: I didn't make them, it did 😞

**Comment by mhutch** [comment]
@garuma fixed!

**Pull request merged**